docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
     ports:
       - "9092"
     environment:
-      KAFKA_ADVERTISED_HOST_NAME: 192.168.99.100
+      KAFKA_ADVERTISED_HOST_NAME: host.docker.internal
```

Comment:
This breaks external access to the broker and I think is only supported by default for docker for windows.

Reply:
docker for macOS is ok

Reply:
A quick check shows it returns an internal to docker host IP (assuming there's some bridge magic to route the traffic back to the host). However, the existing example shows a docker-machine IP, which is routable from the host->container. I believe `host.docker.internal` is for container->host mapping. This is not useful in this scenario, you might as well just leave blank for container hostname. Please correct me if my understanding is incorrect.

Reply:
maybe your are right, I can run multiple brokers when using `host.docker.internal`,  nslookup can get IP addresses like 192.168.65.2, but `host.docker.internal` not work for yahoo/kafka-manager, yahoo/kafka-manager deployed on the docker host